test/PlaygroundTransform/array.swift

```diff
@@ -1,12 +1,27 @@
 // RUN: %empty-directory(%t)
 // RUN: cp %s %t/main.swift
+
+// Build PlaygroundSupport module
 // RUN: %target-build-swift -whole-module-optimization -module-name PlaygroundSupport -emit-module-path %t/PlaygroundSupport.swiftmodule -parse-as-library -c -o %t/PlaygroundSupport.o %S/Inputs/SilentPCMacroRuntime.swift %S/Inputs/PlaygroundsRuntime.swift
-// RUN: %target-build-swift -Xfrontend -playground -o %t/main -I=%t %t/PlaygroundSupport.o %t/main.swift
-// RUN: %target-codesign %t/main
-// RUN: %target-run %t/main | %FileCheck %s
-// RUN: %target-build-swift -Xfrontend -pc-macro -Xfrontend -playground -o %t/main2 -I=%t %t/PlaygroundSupport.o %t/main.swift 
-// RUN: %target-codesign %t/main2
-// RUN: %target-run %t/main2 | %FileCheck %s
+
+// -playground
+// RUN: %target-build-swift -swift-version 5 -Xfrontend -playground -o %t/main5a -I=%t %t/PlaygroundSupport.o %t/main.swift
+// RUN: %target-build-swift -swift-version 6 -Xfrontend -playground -o %t/main6a -I=%t %t/PlaygroundSupport.o %t/main.swift
+
+// -pc-macro -playground
+// RUN: %target-build-swift -swift-version 5 -Xfrontend -pc-macro -Xfrontend -playground -o %t/main5b -I=%t %t/PlaygroundSupport.o %t/main.swift
+// RUN: %target-build-swift -swift-version 6 -Xfrontend -pc-macro -Xfrontend -playground -o %t/main6b -I=%t %t/PlaygroundSupport.o %t/main.swift
```

Comment:
Maybe it's possible to use `RUN: %target-run-simple-swift` instead of build+codesign+run here?

Reply:
Unfortunately not, as that macro removes %t which blows away the PlaygroundSupport module that was just built.

As a separate change, I'd like to look at crafting a PlaygroundTransform-specific macro that takes care of building PlaygroundSupport and linking it, as well as code signing and running.  That should clean things up quite a bit.